<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adjust</class>
 <widget class="QWidget" name="adjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>620</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frBg{
background-image:url(:/main/img/bg.jpg);
}
QFrame#frWhite{
background-color:rgb(255,255,255,230);
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frWhite">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>850</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	font-family:STXihei, &quot;华文细黑&quot;, &quot;Microsoft YaHei&quot;, &quot;微软雅黑&quot;;
font-size:20px;
}
QPushButton{
border-radius:15px;
background:rgb(237,200,199)

}
QPushButton:hover{
background:rgb(255,208,204);
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>450</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>上一步</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>269</x>
       <y>50</y>
       <width>521</width>
       <height>40</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>正在解码，请等待...</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>801</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDecode">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>450</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>解码</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelError">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>460</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
